docs/UserGuide.md

```diff
@@ -235,6 +236,21 @@ Subject: subject
 Current marks out of 100: 100.0
 Classes Attended: 1
 ```
+For any student that exists within the programme's master, archive and recently deleted list, the user cannot add new students with a name-phone number pair that matches that of the existing student.
+
+The following is an example of trying to add a new student that matches the name and phone number of an already existing student.
+
```

Comment:
Perhaps can change to another case which doesn't show the identity of one of our code testers